lib/src/solver/report.dart

```diff
@@ -130,6 +130,32 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    final outdatedPackagesCount = _result.packages.where((id) {
+      final versions = _result.availableVersions[id.name];
+
+      if (versions.any((v) =>
+          v > id.version &&
+          (id.version.isPreRelease == true || v.isPreRelease == false))) {
+        return true;
```

Comment:
`if` can be omitted and we can write:
```dart
(versions.any((v) =>
          v > id.version &&
          (id.version.isPreRelease == true || v.isPreRelease == false)))
```

However, I am not really sure if this is a good practice ...